nearcore/src/config.rs

```diff
@@ -456,8 +452,7 @@ pub struct Config {
     /// For example, setting "use_db_migration_snapshot" to "/tmp/" will create a directory "/tmp/db_migration_snapshot" and populate it with the database files.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub db_migration_snapshot_path: Option<PathBuf>,
-    #[serde(default = "default_enable_rocksdb_statistics")]
-    pub enable_rocksdb_statistics: bool,
+    pub store: near_store::StoreConfig,
```

Comment:
it seems that we removed something (enable_rocksdb_statistics) -- did we set it in any config in any of our jobs?

Reply:
Also - add a comment about this field.